lib/beaker-puppet/install_utils/foss_utils.rb

```diff
@@ -438,11 +437,7 @@ def configure_puppet_on(hosts, opts = {})
         # @api private
         def install_puppet_from_rpm_on( hosts, opts )
           block_on hosts do |host|
-            if host[:type] == 'aio'
```

Comment:
@kevpl how often are we not setting host type to aio? In the code prior to this change, that would enable the old style repos, which are no longer updated at all (we're trying to archive them at some point). How dangerous is it if I make this change?

Reply:
When you say the "older stuff" are you talking about the pc1 repos, or are you talking about the old old repos? I.e., on yum, we have the `products` repos that we used to ship to http://yum.puppetlabs.com/el/6/products/

Reply:
I mean the older things that it could install before these changes. Which, looking at this code path in the method, appear to be just the pc1 repos rather than that & the old `products` repos.